packages/react/src/components/Blockquote/Blockquote.props.ts

```diff
@@ -0,0 +1,4 @@
+export interface BlockquoteProps {
+  quote: string;
+  cite: string;
+}
```

Comment:
There is no need to keep the props in a separate file. I tried to move them inside the component. For the feature, when we relocate the story file to the component directory, we can introduce a new file tree. Until then, let's keep the props inside the component. 

And let's use types instead of interfaces, we can better compose them